WordPress/Classes/Utility/WebKitViewController.swift

```diff
@@ -412,7 +413,7 @@ extension WebKitViewController: WKUIDelegate {
         }
 
         if !ReachabilityUtils.isInternetReachable() {
-            ReachabilityUtils.showAlertNoInternetConnection()
+            ReachabilityUtils.showNoInternetConnectionNotice(message: ReachabilityUtils.noConnectionMessage())
```

Comment:
ℹ️ I now see the effect of what I did in #11303. Oops. 😅

We could probably make the `noConnectionMessage()` a default instead:

```swift
    static func showNoInternetConnectionNotice(message: String = noConnectionMessage()) {
```